scene/resources/particles_material.cpp

```diff
@@ -323,13 +323,13 @@ void ParticlesMaterial::_update_shader() {
 	if (tex_parameters[PARAM_ANGLE].is_valid()) {
 		code += "	float tex_angle = textureLod(angle_texture, vec2(0.0, 0.0), 0.0).r;\n";
 	} else {
-		code += "	float tex_angle = 0.0;\n";
+		code += "	float tex_angle = 1.0;\n";
```

Comment:
Defaulting to 1 radians is somewhat weird.

Reply:
Thank you! Missed that one